tests/vinyldns_tests.bats

```diff
@@ -1,19 +1,13 @@
 load test_helper
 
 @test "groups (when none exist)" {
-  run $ew groups
-
   fixture="$(cat tests/fixtures/groups_none)"
-
-  [ "${output}" = "${fixture}" ]
+  $ew groups | grep "${fixture}"
```

Comment:
@jagadeesh545 Just a nitpick: Is there a reason this change -- which impacts the groups tests -- appears in a pull request pertaining to the addition of a `search-record-sets` command? I just want to be sure I understand :)

Reply:
After updating the fixture to accommodate the default group `globalACLGroup`, I also had to update to the test case to look for text present in the fixture instead of checking for an exact match.